src/components/NodeEditor/NodeEditor.tsx

```diff
@@ -98,41 +182,44 @@ export default class NodeEditor extends React.PureComponent<NodeEditorProps, Nod
     private temporaryButtons?: ButtonSet;
 
     public static contextTypes = {
-        getTypeConfig: getTypeConfigPT
+        getTypeConfig: getTypeConfigPT,
+        getOperatorConfig: getOperatorConfigPT
     };
 
     constructor(props: NodeEditorProps, context: ConfigProviderContext) {
         super(props, context);
 
-        const config = this.context.getTypeConfig(this.determineConfigType());
-
-        this.state = {
-            show: this.props.show || false,
-            config
-        };
+        this.state = this.getInitialState();
+
+        bindCallbacks(this, {
```

Comment:
IMHO this is a much cleaner way to bind callbacks to the component. Lib looks good to me, super small.